nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -75,6 +76,7 @@
 
     // attribute keys
     public static final String SPLIT_LINE_COUNT = "text.line.count";
+    public static final String SPLIT_SIZE = "fragment.size";
```

Comment:
recommend calling this "FRAGMENT_SIZE" to be consistent with the value and other fragment names